Frontend/ui-driver/src/Resource/Localizable/BN.purs

```diff
@@ -1604,7 +1604,7 @@ getBN stringKey =
         PET_FRIENDLY -> "পোষ্য-বান্ধব"
         FROM_WHERE -> "জন্মস্থান"
         DRIVING_SINCE -> "ড্রাইভিং শুরু"
-        WHY_NY -> "আমি কেন অটো চালানোর সিদ্ধান্ত নিলাম?"
+        WHY_NY variant -> "আমি কেন "<> (if variant == "AutoCategory" then getBN AUTO else getBN CAB) <>" চালানোর সিদ্ধান্ত নিলাম?"
```

Comment:
Lets not do this like this, we can make a function to get variant name and pass it wherever we are using this constructor. Other cases like bike ambulance are also there